first-analysis-steps/loki-functors.md

```diff
@@ -72,15 +82,17 @@ p_components_sum(cand)
 ```
 
 {% challenge "Does it make sense?" %}
-Retrieve the momentum magnitude $$p$$ and see if you can get the same answer 
-with the `PX`, `PY`, `PZ` functors.
-Also compute the invariant mass $$m$$ and see if it matches what the `M` 
-functor returned.
+Retrieve the momentum magnitude using functors `PX`, `PY` and `PZ`. 
+
+There is also a specific functor `P` which does the job. Compare the results. 
```

Comment:
The point of this was to use the `momentum()` particle function, not to compare functors. This could have been made clearer, but it's important to establish this parallelism.

Reply:
Ah well, now I see. Just didn't get the point when reading the old version. Will modify to make it clear.